### PR TITLE
The Cratening: Shiptest Edition

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
Resprites the crates into the 3/4ths perspective, but for shiptest! (FEATURING: Special Explorer Crate Resprite!)

This PR also annihilates the goon version of crates.dmi, which was not only previously a dupe of the **regular** crates.dmi, but was in the goon folder for whatever arcane reason.
image

## Why It's Good For The Game
Refer to PR: [#66557](https://github.com/tgstation/tgstation/pull/66557).

All in all, the 3/4ths perspective is something that many servers, including shiptest, are aiming to follow, rather than the previous cabinet perspective that permeated most object sprites in the past. It looks nice too!

## Changelog
:cl: PositiveEntropy
imageadd: Resprites most crates and the trash cart!
del: Removes crates.dmi from the goon folder.
/:cl:

### CODE REVIEW

**Review by triplezeta — APPROVED** [review]
Okay! :smiley:

File: code/modules/ruins/lavalandruin_code/elephantgraveyard.dm [hunk 1]
```diff
@@ -109,7 +109,7 @@
 /obj/structure/closet/crate/grave
 	name = "burial mound"
 	desc = "An marked patch of soil, showing signs of a burial long ago. You wouldn't disturb a grave... right?"
-	icon = 'goon/icons/obj/crates.dmi'
+	icon = 'icons/obj/crates.dmi'
```
Comment (Fikou) [inline code/modules/ruins/lavalandruin_code/elephantgraveyard.dm]: this is inherited from parent

Comment (Fikou) [inline code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:112]: this is inherited from parent

File: code/modules/ruins/lavalandruin_code/elephantgraveyard.dm [hunk 2]
```diff
@@ -198,7 +198,7 @@
 /obj/structure/closet/crate/grave/lead_researcher
 	name = "ominous burial mound"
 	desc = "Even in a place filled to the brim with graves, this one shows a level of preperation and planning that fills you with dread."
-	icon = 'goon/icons/obj/crates.dmi'
+	icon = 'icons/obj/crates.dmi'
```
Comment (Fikou) [inline code/modules/ruins/lavalandruin_code/elephantgraveyard.dm]: this too

Comment (Fikou) [inline code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:201]: this too

**Review by Fikou — APPROVED** [review]
looks good!

**Pull request merged**